modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -64,7 +72,9 @@ endif::[]
 
 2. Egress router for OVN-Kubernetes supports only redirect mode.
 
-3. IPv6 is supported only on bare metal, vSphere, {ibm-power-name}, {ibm-z-name}, and Red Hat OpenStack clusters.
+3. IPv4-primary dual-stack networking on bare-metal, {vmw-full}, {ibm-power-name}, {ibm-z-name}, and {rh-openstack} platforms.
```

Comment:
Since 4.13 we seem to track this in the code now with `allowV6Primary` flag in 

https://github.com/openshift/installer/blob/release-4.13/pkg/types/validation/installconfig.go#L241